src/service/data_products/genome_attributes.py

```diff
@@ -222,18 +228,51 @@ def _remove_keys(doc):
 _FLD_LIMIT = "limit"
 
 
-# At some point we're going to want to filter/sort on fields. We may want a list of fields
-# somewhere to check input fields are ok... but really we could just fetch the first document
-# in the collection and check the fields 
+@_ROUTER.get(
+    "/meta",
+    response_model=col_models.ColumnarAttributesMeta,
+    description=
+"""
+Get metadata about the genome attributes table including column names, type,
+minimum and maximum values, etc.
+""")
+async def get_genome_attributes_meta(
+    r: Request,
+    collection_id: str = PATH_VALIDATOR_COLLECTION_ID,
+    load_ver_override: common_models.QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = None,
+    user: kb_auth.KBaseUser = Depends(_OPT_AUTH)
+    ):
+    storage = app_state.get_app_state(r).arangostorage
+    _, load_ver = await get_load_version(storage, collection_id, ID, load_ver_override, user)
+    doc = await get_collection_singleton_from_db(
+            storage,
+            names.COLL_GENOME_ATTRIBS_META,
+            collection_id,
+            load_ver,
+            bool(load_ver_override)
+    )
+    doc[col_models.FIELD_COLUMNS] = [col_models.AttributesColumn.model_construct(**d)
```

Comment:
cannot find model_construct method. Is this for later PR?

Reply:
`model_construct` is a pydantic method